tests/Microsoft.Identity.Test.Unit/Microsoft.Identity.Test.Unit.csproj

```diff
@@ -12,6 +12,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\src\client\Microsoft.Identity.Client.Brokers\Microsoft.Identity.Client.Broker.csproj" />
+    <ProjectReference Include="..\..\src\client\Microsoft.Identity.Client.Broker\Microsoft.Identity.Client.Broker.csproj" />
```

Comment:
Somehow the same project ref added twice.

Reply:
not needed